dwave/system/temperatures.py

```diff
@@ -0,0 +1,488 @@
+    distributions, particularly those with rare non-thermal high energy 
+    excitations. When the distribution is Boltzmann, both methods yield the same
+    temperature up to sampling error. Maximum likelihood estimation generalizes
+    to temperature estimates over samples drawn from arbitrary Hamiltonians,
+    pseudo-likelihood estimation is efficient for arbitrary Hamiltonians.
+    
+    The Advantage QPU has known deviations from an ideal Boltzmann sampler such,
+    as flux noise, please refer to literature and documentation for more 
+    details. Estimators accounting for such non-idealities may produce
+    different results.
+
+    For statistical efficiency, and in the case of QPU to avoid poor performance
+    [due to noise and calibration non-idealities], it can be useful to submit 
+    problems with energies comparable to 1/Temperature. The default value
+    is based upon the single-qubit freeze-out hypothesis: B(s*)/kB T, for the 
+    online system at temperature of 12mK, freeze-out value s*.
+
+    We need to probe problems at an energy scale where excitations
+    are common, but at which a sampler is not dominated by noise (precision).
+    For Advantage_system1.1 the longitudinal field value at 
+    freeze-out B(s=0.612) = 3.91GHz, and operational temperature of
+    15.4mK, imply an effective temperature for single qubits of
+    T_guess = B(s^*)/[2 k_B T] ~ 6 which is used by default; and is 
+    sufficiently close for related online systems.
+
+    Args:
+        sampler (:class:`dimod.Sampler`, optional, default=\ :class:`~dwave.system.samplers.DWaveSampler`\ ``(client="qpu")``):
+            A D-Wave sampler. 
+
+        num_reads (int, optional):
+            Number of reads to use. Default is 100 if not specified in 
+            sampler_params.
+
+        seed (int, optional):
+            Seeds the problem generation process. Allowing reproducibility
+            from pseudo-random samplers.
+
+        T_guess (int, optional, default = 6.1):
+            Determines the range of external fields (h_i) probed for temperature
+            inference; an accurate choice raises the efficiency
+            of the estimator. An inappropriate choice may lead to
+            pathological behaviour. Default is based on D-Wave Advantage 
+            processor temperature and energy scales, and is also suitable for 
+            D-Wave 2000Q processor inference.
+        
+        sampler_params (dict, optional):
+            Any additional non-defaulted sampler parameterization. If num_reads
+            is a key, must be compatible with num_reads argument.
+
+    Returns:
+        float:
+            The effective temperature describing single qubit problems in an
+            external field.
+
+    See also:
+        https://doi.org/10.3389/fict.2016.00023
+    
+    Examples:
+       Draw samples from the default DWaveSampler(), and establish the temperature
```

Comment:
"the default DWaveSampler()" --> same as before about a "default DWaveSampler()" being misleading, perhaps "from a DWaveSampler()"